content/docs/intro/console/_index.md

```diff
@@ -1,7 +1,13 @@
 ---
-title: Pulumi Cloud Console
-
-aliases: [/docs/reference/service]
+title: Pulumi Console
+menu:
+  intro:
+    identifier: console
+    weight: 1
+
+aliases:
+- /docs/reference/service
```

Comment:
FWIW, I prefer this syntax for lists. I see we're using both, so might be good to go with one or the other.

Reply:
Using the list style across the board (even when there's a single item) sounds good to me.

Reply:
I'll clean this up subsequently.